src/components/qas/questions/qaCalculator/qaCalculatorIcons.jsx

```diff
@@ -21,10 +21,9 @@ QACalculatorIcon.propTypes = {
 
 export const QACalculatorIconUnit = ({ unit, tiny, className }) => {
```

Comment:
Typically we've been writing React component filenames in CapitalCamelCase and reserving lowercaseCamelCase for vanilla js files (index.jsx files not withstanding).

Reply:
Updated 👍